core/app_modules/cal.js

```diff
@@ -21,6 +28,8 @@ module.exports.addCalendarEvent = co.wrap(function*(username, event, user){
   }
   if (!isOverlap) {
     yield client.nonQuery('INSERT INTO calendar("user", "time", room, duration, title)VALUES ($1, $2, $3, $4, $5);', [user, event.time, event.room, event.duration, event.title]);
+    user_info = yield get_user_info(user);
+    email.roomConfirmation(user_info, event.time, event.room)

```

Comment:
email.roomConfirmation expects us to know a lot about the user. Mostly it will crash and burn (no matter what I do) if an email doesn't exist for said user. What should be the default behavior if email isn't provided for a user? Just barf, or send an email to ENV.EMAIL that says "YO this person doesn't have an email that we know of, so no confirmation was sent"